lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb

```diff
@@ -546,15 +546,20 @@ def self.update_release(api_token, owner_name, app_name, release_id, release_not
       end
 
       # updates release metadata
-      def self.update_release_metadata(api_token, owner_name, app_name, release_id, dsa_signature)
-        return if dsa_signature.to_s == ''
+      def self.update_release_metadata(api_token, owner_name, app_name, release_id, dsa_signature = '', ed_signature = '')
+        return if dsa_signature.to_s == '' && ed_signature.to_s == ''
 
         url = "v0.1/apps/#{owner_name}/#{app_name}/releases/#{release_id}"
         body = {
-          metadata: {
-            dsa_signature: dsa_signature
-          }
+          metadata: {}
         }
+        
+        if dsa_signature.to_s != ''
+          body["metadata"]["dsa_signature"] = dsa_signature
+        end
+        if ed_signature.to_s != ''
+          body["metadata"]["ed_signature"] = ed_signature
+        end
```

Comment:
This code is not valid. `metadata` in `body` is a symbol (`:metadata`), you're indexing here as a string `"metadata"`.
```suggestion
        if dsa_signature.to_s != ''
          body[:metadata]["dsa_signature"] = dsa_signature
        end
        if ed_signature.to_s != ''
          body[:metadata]["ed_signature"] = ed_signature
        end
```